zuko-ui/src/components/AboutHero/AboutHero.jsx

```diff
@@ -0,0 +1,24 @@
+import styles from './AboutHero.module.scss'
+
+export default function AboutHero({ header = "O nama", src, alt = "Missing image", text1, text2, description }) {
+    const texts = [
```

Comment:
We don't need array for this use-case, use can create constants:

const text1Default= "Zensko udruzenje kol...."
const text2Default = "Vremenom su se ....."

without map function